examples/runners/wgpu/src/compute.rs

```diff
@@ -69,24 +74,29 @@ pub fn start(options: &Options) {
         module: &module,
         entry_point: "main_cs",
     });
-
-    let buf = device.create_buffer(&wgpu::BufferDescriptor {
+    let size = (src.len() * std::mem::size_of::<u32>()).try_into().unwrap();
+    let staging_buffer = device.create_buffer(&wgpu::BufferDescriptor {
```

Comment:
I think this should be called `readback_buffer`. "staging" usually implies that it's being used as an intermediate to copy data to cpu side *before* copying to a gpu only buffer.